GameData/RP-1/Programs/Programs.cfg

```diff
@@ -482,6 +487,59 @@ RP0_PROGRAM
 		Fast = 900
 	}
 }
+RP0_PROGRAM
+{
+	name = CommercialApplications2
+	title = Advanced Commercial Applications
+	description = Geosynchronous, Geostationary, Molniya, and Tundra orbits have the potential to improve satellite coverage and simplify ground antenna tracking. This program tasks you with testing these orbits, launching satellites into them for commercial customers, and creating a 3 or 4 satellite Geostationary or Molniya Network for communication.
+	requirementsPrettyText = Complete the Early Commercial Applications Program
+	objectivesPrettyText = Complete a Geostationary or Molniya Communications Network.
+	nominalDurationYears = 3
+	baseFunding = 800000
+	fundingCurve = MildBackloadedFundingCurve
+	repDeltaOnCompletePerYearEarly = 420
+	repPenaltyPerYearLate = 420
```

Comment:
Not sure why these values are set so high. I feel like 270 would be more appropriate.